woocommerce/woocommerce
# TT3 compatibility

### All Submissions:






### Changes proposed in this Pull Request:



This work builds on the great cleanup done by Rubik/Kirigami earlier this year. I did a tiny cleanup of colors from blocktheme.scss, where I moved the unnecessary definitions to TT2 theme's CSS to retain the same styling for TT2. 

Besides, did a quick analysis and comparison for all CSS rules and added the relevant ones from TT2 styling to TT3. Added also the WP class to buttons to enable all the style definitions that come with WP 6.1 and TT3.

Layout-wise, it's very similar to TT2. The colors got tricky here and there as some styles have effectively only 2 colors while some define 5 distinct colors, but I tried to make it work for all styles.

Style colors overview:
![TT3colors](https://user-images.githubusercontent.com/2207451/197823153-b07f8f82-a5f5-492f-a944-92241bd5f89b.jpg)







### How to test the changes in this Pull Request:



1. Install WP 6.1 RC and TT3
2. Install WC and activate this branch
3. Explore all the important screens: 

- Shop, 
- Single product for simple product
- Single product for variable products
- Examine gallery when having multiple images, photoswipe fullscreen image
- Check the Sale! badge looks reasonable in Shop and in single product page
- Check the tabs: Additional info/Details/Reviews
- Make sure the reviews are visible, rating looks reasonable
- Cart (try with or without shipping options set up), adding coupon
- Checkout
- Thank you page after submitting the order
- Log in to My account, with or without registration
- My account page
- Check out My Account submenus
- Test store notices (Added to cart on single product page, Error on the checkout page, Add coupon on the checkout page, Log in on the checkout page)
- Test store-wide "demo store" notice
- Check this in different viewport sizes
- Check all of this with different browsers
- Check all of this with each style in TT3 (Appearance > Editor > Click on Styles (half moon icon to the top right) and switch to another one)
![Screenshot 2022-10-25 at 17 49 45](https://user-images.githubusercontent.com/2207451/197823191-114bcbd7-dda0-4748-b7e6-20dc654f31b0.png)




### Other information:




### FOR PR REVIEWER ONLY:
